alonet/detr/criterion.py

```diff
@@ -6,6 +6,67 @@
 import aloscene
 
 
+def dice_loss(inputs: torch.Tensor, targets: torch.Tensor, num_boxes: int) -> torch.Tensor:
```

Comment:
Theses new extensions should be in the `detr_panoptic` instead of in the `detr` folder. You should have a DetrPanopticCriterion that inhrt this class

Reply:
Done